app/views/matchups/_matchup.html.erb

```diff
@@ -1,22 +1,13 @@
-<div class="card mb-2">
-  <div class="card-body">
-    <p class="card-text">
-      <% if matchup.round == 1 %>
-        <%= render "entries/entry", entry: matchup.option_a %>
-      <% else %>
-        <%= matchup.round %>
-        Winner from previous round
-      <% end %>
-    </p>
-    <p class="card-text">
-      <% if matchup.round == 1 %>
-        <%= render "entries/entry", entry: matchup.option_b %>
-      <% else %>
-        <%= matchup.round %>
-        Winner from previous round
-      <% end %>
-    </p>
+<% if matchup.bracket.status == "created" %>
```

Comment:
Since status is an enum you get [`created?` for free](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)!

```suggestion
<% if matchup.bracket.created? %>
```